packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx

```diff
@@ -41,6 +41,10 @@ export class ColorPickerBasicExample extends React.Component<{}, IBasicColorPick
           styles={colorPickerStyles}
           // The ColorPicker provides default English strings.
           // If your app is localized, you MUST provide the `strings` prop with localized strings.
+          strings={{
+            alphaAriaLabel: 'Alpha Slider: Use left and right arrow keys to change value, hold shift for a larger jump',
```

Comment:
Probably good to add something to the comment above explaining why only these strings are here, like: "You can also override specific strings to provide more detailed instructions."